src/elements/featured-videos.html

```diff
@@ -0,0 +1,330 @@
+<link rel="import" href="../../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../../bower_components/plastic-image/plastic-image.html">
+<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
+<link rel="import" href="../bower_components/polymer/lib/mixins/gesture-event-listeners.html">
+<link rel="import" href="../redux-mixin.html">
+<link rel="import" href="flex-layout-attr.html">
+<link rel="import" href="video-dialog.html">
+
+<dom-module id="featured-videos">
+  <template>
+    <style is="custom-style" include="shared-styles flex flex-alignment"></style>
+
+    <style>
+       :host {
+        display: block;
+      }
+
+      .videos-wrapper {
+        padding: 0;
+      }
+
+      .header {
+        margin: 24px auto;
+        padding: 0 16px;
+      }
+
+      .title {
+        font-size: 24px;
+      }
+
+      .icon-label {
+        margin-left: 10px;
+      }
+
+      .video-icon {
+        margin-left: 0;
+        padding-left: 0;
+      }
+
+      .video-list {
+        position: relative;
+        margin: 0 auto;
+        max-width: 1280px;
+        height: 250px;
+        overflow: hidden;
+        margin-left: -15px;
+      }
+
+      .videos {
+        transition: all 0.4s cubic-bezier(0, 0, 0.2, 1);
+        will-change: transition;
+        backface-visibility: hidden;
+        transform: translateX(0px);
+      }
+
+      .video-item {
+        width: 315px;
+        min-width: 315px;
+        height: 213px;
+        box-sizing: border-box;
+        position: relative;
+        padding: 0 15px;
+        box-sizing: border-box;
+      }
+
+      .preview-image {
+        display: block;
+        cursor: pointer;
+        width: 280px;
+        min-width: 270px;
+        height: 177px;
+        border-radius: 5px;
+        margin-bottom: 10px;
+        overflow: hidden;
+        box-sizing: border-box;
```

Comment:
Unnecessary as defined in shared-styles
https://github.com/gdg-x/hoverboard/blob/hoverboard-v2/src/elements/shared-styles.html#L32